### ISSUE TITLE
max-height on cropper-canvas

### ISSUE THREAD

**Issue by boustanihani** [fengyuanchen/cropper#238]
I am using the following options:
```javascript
    toBeCropped.cropper({
        strict: true,
        background: false,
        zoomable: false,
        autoCropArea: 0.8,
        aspectRatio: 16 / 9,
    });
```
I disabled zooming and I want the user to see the whole image at once without the need of scrolling so I am also setting ```max-height``` on the "toBeCropped" image in respect to the window height but this is not getting applied to the final ```cropper-canvas``` or the contained image...

Is there a way to apply ```max-height``` to the ```cropper-canvas``` or the contained image ??

Here is the original image:
![original-image](https://cloud.githubusercontent.com/assets/4227548/6657688/508a7f5a-cb53-11e4-9e69-efae8612be8b.jpg)

And here is the current situtation, **the user sees a small part of the image and the rest is invisible**:
![crop-moal](https://cloud.githubusercontent.com/assets/4227548/6657692/6b89efb6-cb53-11e4-9648-5c8f8a39af63.png)

**Comment by boustanihani** [comment]
**And this is how the image should be displayed**:

![crop-modal-2](https://cloud.githubusercontent.com/assets/4227548/6657801/d11bab32-cb56-11e4-9121-8e773ab7ceae.png)

**Comment by fengyuanchen** [comment]
In your case, you should set `strict: false`

**Comment by boustanihani** [comment]
```strict: false``` is not an option because it allows the crop box to be outside of the canvas (which is something that makes no sense in my case), why cannot I simply use strict and also display the image toBeCropped the way I like (with ```max-height``` adjusted to the screen size) ?

![screenshot - 16 03 2015 - 23 10 14](https://cloud.githubusercontent.com/assets/4227548/6677856/244a93bc-cc34-11e4-8b19-4838a40a72ff.jpg)

**Comment by AlexRieger** [comment]
I see that you seem to be against any improvement of the strict mode since you are cloing down the issues without any interest in discussion. Do you mind if I ask why? You said concerning my issue (#242) that I should stick to the old version. I think is a bad advice since it will keep me away from any new features.
Maybe I just don't get it, but I don't see any use for the strict mode when it behaves like that, can you tell me what I (or me and Mr. Isse #338) are dong wrong here? How do we get a picture that does not match the aspect ratio of the canvas to see in full in strict mode? Or is this the Problem? Do we have to set the aspect ratio of the canvas to the one of the picture to do the trick?

**Comment by boustanihani** [comment]
@AlexRieger You spoke my mind :+1: 

I hope we get an answer :)

**Comment by rossbearman** [comment]
I have to agree here, strict mode seems broken in it's current form. Being unable to allow users to crop an image of any aspect ratio down, while still restricting them to only crop within the area of the image, doesn't seem to make any sense. Surely this is a very common use case for this plugin?

**Comment by boustanihani** [comment]
@AlexRieger This seems to have been fixed with **v0.9.2**, will test the coming days:
https://github.com/fengyuanchen/cropper/releases/tag/v0.9.2

**- Improve strict mode to show full image**
